habitat_baselines/config/rearrange/gala_kinematic_ddppo.yaml

```diff
@@ -1,5 +1,5 @@
-TENSORBOARD_DIR: "/checkpoint/eundersander/gala_kinematic/tb/gala_kinematic_ddppo"
-CHECKPOINT_FOLDER: "/checkpoint/eundersander/gala_kinematic/ckpt/gala_kinematic_ddppo"
+TENSORBOARD_DIR: "data/tb/gala_kinematic_ddppo"
```

Comment:
On the FAIR cluster, I guess we aren't supposed to be writing/checkpointing to our home directory (the disk write IO is kinda slow or something). I wish I knew the correct way to do this.

Reply:
I think writing to /checkpoint is correct.

Reply:
Ok. Do you wanna revise this line before we merge this PR?

I wish we could do `/checkpoint/{username}/gala_kinematic/ckpt/gala_kinematic_ddppo`, where Hab-lab auto-inserts your actual username.